frontend/src/components/TaskCard.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import {
+  Box,
+  Card,
+  Typography,
+  Avatar,
+  Chip,
+  AvatarGroup
+} from '@mui/material'
+import PropTypes from 'prop-types'
+import PriorityIcon from './PriorityIcon.js'
+import './TaskCard.sass'
+
+
+const TaskCard = (props) => {
+  return (
+    <div className='TaskCard'>
+      <Card className='.MuiCard-root'>
+        <Typography>
+          {props.taskLabel}
+        </Typography>
+        <Box className='Box-categories'>
+          {props.taskCategories.map((taskCategory) => (
+            <Chip key={taskCategory[0]} label={taskCategory[0]} sx={{ color: taskCategory[1], bgcolor: taskCategory[2] }} size='small' />
+          ))}
+        </Box>
+        <Box className='Box'>
+          <Box className='Box-priority'>
+            <PriorityIcon taskPriority={props.taskPriority} />
+            {props.taskPoints ? <Avatar className='Box-priority-avatar'>{props.taskPoints}</Avatar> : null}
```

Comment:
Tak nazwa klasy `Box-priority-avatar` jest bardzo myląca, bo spodziewałbym się, że to będzie `<Box/>`. Nie to miałem na myśli rozumiejąc podklasy.

To powinno być np. `Avatar-priority` i potem w Sass:
```sass
.Box
  .Box-priority
    .Avatar-priority
      jakies: wlasnosci
```

Reply:
Albo po prostu
```sass
.Box
  .Box-priority
    .priority-avatar
      jakies: wlasnosci
```